README.md

```diff
@@ -8,3 +8,4 @@ It is a little bit hard to calculate loan properly. Therefore, I create a small
 ## How to install?
 
 The code is compatible with both `Python2`/`Python3`. In order to run it, you need to install python and run code as `python credit_calculator.py` changing parameters based on the loan.
+You must specify the credit sum as a first arg and the monthly payment as the second arg.
```

Comment:
@Ghaust Can you add example of command like call and example of output

Reply:
@Ghaust
Here, I expect it to be more like `python credit_calculator.py --sum 3000 --payment 100` and example of the output. If it is too much for one PR, I can decompose the issue in several.